addons/pos_restaurant/static/tests/tours/pos_restaurant_tour.js

```diff
@@ -402,6 +403,54 @@ registry.category("web_tour.tours").add("PreparationPrinterContent", {
         ].flat(),
 });
 
+registry.category("web_tour.tours").add("ComboSortedPreparationReceiptTour", {
+    checkDelay: 50,
+    steps: () =>
+        [
+            Chrome.startPoS(),
+            Dialog.confirm("Open Register"),
+            FloorScreen.clickTable("5"),
+            ProductScreen.clickDisplayedProduct("Office Combo"),
+            combo.select("Combo Product 2"),
+            combo.select("Combo Product 4"),
+            combo.select("Combo Product 6"),
+            Dialog.confirm(),
+            ProductScreen.clickDisplayedProduct("Office Combo"),
+            combo.select("Combo Product 1"),
+            combo.select("Combo Product 5"),
+            combo.select("Combo Product 8"),
+            Dialog.confirm(),
+            ProductScreen.clickOrderButton(),
+            {
+                trigger: ".render-container",
+                run: () => {
+                    const orderLines = [
+                        ...document.querySelectorAll(".render-container .orderline"),
+                    ];
+                    const orderLinesInnerText = orderLines.map((orderLine) => orderLine.innerText);
+                    const expectedOrderLines = [
+                        "Office Combo",
+                        "Combo Product 2",
+                        "Combo Product 4",
+                        "Combo Product 6",
+                        "Office Combo",
+                        "Combo Product 1",
+                        "Combo Product 5",
+                        "Combo Product 8",
+                    ];
+                    for (let i = 0; i < orderLinesInnerText.length; i++) {
+                        if (!orderLinesInnerText[i].includes(expectedOrderLines[i])) {
+                            throw new Error("Order line mismatch");
+                        }
+                    }
```

Comment:
We should stop using render-container.

See https://github.com/odoo/odoo/pull/197212 :/